runtime/near-vm-runner/Cargo.toml

```diff
@@ -16,6 +16,9 @@ This crate implements the specification of the interface that Near blockchain ex
 borsh = "0.7.1"
 wasmer-runtime = { version="0.17.1", features = ["default-backend-singlepass"], default-features = false, package = "wasmer-runtime-near" }
 wasmer-runtime-core = {version = "0.17.1", package = "wasmer-runtime-core-near" }
+wasmer = { git = "https://github.com/wasmerio/wasmer", rev="b4d2d8ee804a45850af2b3e012fdbbbdcb1185cb", optional = true }
```

Comment:
Is 1.0 not release to crates.io yet?

Reply:
It's released but i need a patch https://github.com/wasmerio/wasmer/pull/2026, it might be released in 1.0.2

Reply:
Shalln't we ask them to publish it and depend on published version?

Reply:
Asked